<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrudZone</class>
 <widget class="QMainWindow" name="CrudZone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1375</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Administración de zonas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 134, 137)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>1250</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1250</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabInsertZone">
     <attribute name="title">
      <string>Alta de zonas</string>
     </attribute>
     <widget class="QGroupBox" name="gbPrimary">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>1101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lneIdZona">
       <property name="geometry">
        <rect>
         <x>870</x>
         <y>10</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbName">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbState">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Estado</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cboState">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>60</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbMunicipio">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Municipio</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cboMunicipio">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>61</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Id Zona</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pbOut">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>360</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/Images/door_out.ico</normaloff>:/Images/door_out.ico</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pbActive">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Activar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbInsert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>360</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbImage">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>280</y>
        <width>311</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images2.qrc">:/littleleatherbackturtlesrogerleguen.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>300</y>
        <width>251</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(167, 135, 100);</string>
      </property>
      <property name="text">
       <string>Es importante llevar a cabo
un control sobre las zonas
donde las tortugas abundan
para así tener presente
que estas deberán ser
monitoreadas con mayor
frecuencia.</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabQueryZone">
     <attribute name="title">
      <string>Busqueda de zonas</string>
     </attribute>
     <widget class="QPushButton" name="pbActiveQuery">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>460</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Activar</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gbPrimaryQuery">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>1071</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="gbSecundaryQuery">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>931</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lbQuery">
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Busqueda</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lnQuery">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Raleway</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbCritery">
          <property name="font">
           <font>
            <family>Raleway</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Criterio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboCritery">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Raleway</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tvQuery">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>931</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pbOut2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>460</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/Images/door_out.ico</normaloff>:/Images/door_out.ico</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lbTS">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>440</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="images2.qrc">:/TunderSoft.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lbBackGif">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1375</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>lbBackGif</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images2.qrc"/>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbOut</sender>
   <signal>clicked()</signal>
   <receiver>CrudZone</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
